frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { ObjectDataReductionBody } from './object-data-reduction-card-body';
+import { ObjectDataReductionItem } from './object-data-reduction-card-item';
+import './object-data-reduction-card.scss';
+
+const ObjectDataReductionQueries = {
```

Comment:
Can you move it to `queries.ts` as per https://github.com/openshift/console/pull/1935/files#r300984541

Reply:
@rawagner sure I was actually planning to do in a seperate PR for all components with other changes